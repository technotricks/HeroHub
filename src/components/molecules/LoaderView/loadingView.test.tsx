import 'react-native';
import React from 'react';

import {it, expect, describe} from '@jest/globals';

import {create} from 'react-test-renderer';
import LoadingView from '.';

describe('LoadingView component should render correctly', () => {
  const view = create(<LoadingView />);

  it('snapshot', () => {
    expect(view).toMatchSnapshot();
  });
});
