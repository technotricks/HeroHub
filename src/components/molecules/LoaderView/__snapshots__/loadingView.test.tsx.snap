// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorView component should render correctly snapshot 1`] = `
<RCTSafeAreaView
  style={
    {
      "flexBasis": 0,
      "flexGrow": 1,
      "flexShrink": 1,
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "height": "100%",
        "justifyContent": "center",
      }
    }
  >
    <ActivityIndicator
      color="red"
      size="large"
    />
  </View>
</RCTSafeAreaView>
`;

exports[`LoadingView component should render correctly snapshot 1`] = `
<RCTSafeAreaView
  style={
    {
      "flexBasis": 0,
      "flexGrow": 1,
      "flexShrink": 1,
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "height": "100%",
        "justifyContent": "center",
      }
    }
  >
    <ActivityIndicator
      color="red"
      size="large"
    />
  </View>
</RCTSafeAreaView>
`;
